File: src/main/java/kr/co/bookand/backend/account/domain/RevokeAccount.java
package kr.co.bookand.backend.account.domain;

import lombok.*;

import javax.persistence.*;

@Entity
@Getter
@AllArgsConstructor
@NoArgsConstructor(access = AccessLevel.PROTECTED)
@Builder
public class RevokeAccount {

    @Id
    @GeneratedValue
    @Column(name = "revoke_account_id")
    private Long id;

    private String reason;

    @Enumerated(EnumType.STRING)
    private RevokeType revokeType;

    private Long accountId;
}

File: src/main/kotlin/kr/co/bookand/backend/account/domain/RevokeAccount.kt
package kr.co.bookand.backend.account.domain

import kr.co.bookand.backend.common.domain.BaseEntity
import javax.persistence.*

@Entity
class RevokeAccount(
    @Id
    @GeneratedValue
    @Column(name = "revoke_account_id")
    var id: Long? = 0,

    var reason: String,

    @Enumerated(EnumType.STRING)
    var revokeType: RevokeType,

    var accountId: Long
): BaseEntity() {
}